docs/src/content/docs/ko/guides/authoring-content.md

```diff
@@ -0,0 +1,210 @@
+---
+title: 마크다운으로 콘텐츠 작성
+description: Starlight가 지원하는 Markdown 구문의 개요입니다.
+---
+
+Starlight는 `.md` 파일에서 제목 및 설명과 같은 메타데이터를 정의하기 위해 [Markdown](https://daringfireball.net/projects/markdown/)의 모든 구문과 frontmatter [YAML](https://dev.to/paulasantamaria/introduction-to-yaml-125f)을 지원합니다.
+
+해당 파일 형식을 사용하는 경우 Markdown 지원 및 사용법이 다를 수 있으므로 [MDX 문서](https://mdxjs.com/docs/what-is-mdx/#markdown) 또는 [Markdoc 문서](https://markdoc.dev/docs/syntax)를 확인하세요.
+
+## 인라인 스타일
+
+텍스트는 **굵게**, _기울임꼴_ 또는 ~취소선~으로 표시할 수 있습니다.
+
+```md
+텍스트는 **굵게**, _기울임꼴_ 또는 ~취소선~으로 표시할 수 있습니다.
+```
+
+[다른 페이지로 링크](/ko/getting-started)할 수 있습니다.
+
+```md
+[다른 페이지로 링크](/ko/getting-started)할 수 있습니다.
+```
+
+백틱을 사용하여 `인라인 코드`를 강조 표시할 수 있습니다.
+
+```md
+백틱을 사용하여 `인라인 코드`를 강조 표시할 수 있습니다.
+```
+
+## 이미지
+
+Starlight의 이미지는 [Astro에 내장된 최적화된 자산 지원](https://docs.astro.build/en/guides/assets/)을 사용합니다.
```

Comment:
Although it’s not translated yet, it’s probably better to link to the `/ko/` version of this page in the Astro docs:

```suggestion
Starlight의 이미지는 [Astro에 내장된 최적화된 자산 지원](https://docs.astro.build/ko/guides/assets/)을 사용합니다.
```